src/main/java/net/mcreator/ui/WindowTitleHelper.java

```diff
@@ -30,24 +30,23 @@ public static String getWindowTitle(MCreator mcreator) {
 		String appendix = "";
 		if (mcreator.mcreatorTabs.getCurrentTab() != null && mcreator.mcreatorTabs.getCurrentTab()

```

Comment:
The refactor of this file has nothing to do with the PR. Please keep such changes (anything that does not connect to the actual PR intent) out of the (already ~~too~~ big) PRs.

Reply:
Move this (with `getPathInWorkspace()`) to smaller PR like with #3499?

Reply:
Yes, that would be a good idea. Also make getPathInWorkspace throw runtimeexception if used on a file outside the workspace as this is either a bug or something trying to access things it should not access

Reply:
✔️ Fixed